src/mcp/README.md

```diff
@@ -37,10 +37,11 @@ In `.cursor/mcp.json` in your workspace directory, add the Firebase MCP server:
 | Tool Name                           | Feature Group | Description                                                                                                                                                                                                                                                       |
 | ----------------------------------- | ------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | firebase_get_project                | core          | Retrieves information about the currently active Firebase project.                                                                                                                                                                                                |
-| firebase_use_project                | core          | Select a Firebase Project to use for subsequent tool calls.                                                                                                                                                                                                       |
 | firebase_list_apps                  | core          | Retrieves apps registered in the current Firebase project.                                                                                                                                                                                                        |
 | firebase_get_sdk_config             | core          | Retrieves the Firebase SDK configuration information for the specified platform. You must specify either a platform or an app_id.                                                                                                                                 |
 | firebase_consult_assistant          | core          | Send a question to an AI assistant specifically enhanced to answer Firebase questions.                                                                                                                                                                            |
+| firebase_get_environment            | core          | Retrieves information about the current Firebase environment including current user, project directory, active project, and more.                                                                                                                                 |
```

Comment:
Should we get rid of `set_project_directory` below in favor of `firebase_update_environment`?